pkg/models/user_auth.go

```diff
@@ -36,6 +36,15 @@ type ExternalUserInfo struct {
 	IsDisabled     bool
 }
 
+type LoginInfo struct {
+	AuthModule   string
+	User         *User
+	ExternalUser ExternalUserInfo
+	Query        *LoginUserQuery
```

Comment:
Do we need to pass the password here?

Other than the IP address, what additional information do we get from sending the query in the hook?

Reply:
The IP address can easily be retrieved from the request (and it's sent through the hook) so yes, I guess we only need the username used in the login form. I'll change that!